instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -180,11 +190,12 @@ func (c *config) handleRPC(ctx context.Context, rs stats.RPCStats, isServer bool
 				),
 			)
 		}
-	case *stats.OutTrailer:
```

Comment:
Why is this removed?

Reply:
It is not doing anything (the statement is empty), so it's covered by `default`.